### PR TITLE
Jkmarx/solr user files bug

### LINKED ISSUE #2623: RuntimeError in files API
* Specific code commit: v1.6.2
* Version of the web browser and OS: unknown
* Environment where the error occurred: AWS

### Steps to reproduce
@gmnelson: "I went to the User Files browser while logged in as admin (so lots of files)"

### Observed behavior
```
Internal Server Error: /api/v2/files/
Traceback (most recent call last):
  File "/home/ubuntu/.virtualenvs/refinery-platform/lib/python2.7/site-packages/django/core/handlers/base.py", line 111, in get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "/home/ubuntu/.virtualenvs/refinery-platform/lib/python2.7/site-packages/django/views/decorators/csrf.py", line 57, in wrapped_view
    return view_func(*args, **kwargs)
  File "/home/ubuntu/.virtualenvs/refinery-platform/lib/python2.7/site-packages/django/views/generic/base.py", line 69, in view
    return self.dispatch(request, *args, **kwargs)
  File "/home/ubuntu/.virtualenvs/refinery-platform/lib/python2.7/site-packages/rest_framework/views.py", line 466, in dispatch
    response = self.handle_exception(exc)
  File "/home/ubuntu/.virtualenvs/refinery-platform/lib/python2.7/site-packages/rest_framework/views.py", line 463, in dispatch
    response = handler(request, *args, **kwargs)
  File "/srv/refinery-platform/refinery/user_files_manager/views.py", line 56, in get
    solr_response = _get_solr(request.query_params, request.user.id)
  File "/srv/refinery-platform/refinery/user_files_manager/views.py", line 74, in _get_solr
    return search_solr(solr_params, 'data_set_manager')
  File "/srv/refinery-platform/refinery/data_set_manager/utils.py", line 854, in search_solr
    'Expected Solr JSON, not: {}'.format(full_response.content)
RuntimeError: Expected Solr JSON, not:

Request repr():
<WSGIRequest
path:/api/v2/files/,
GET:<QueryDict: {u'sort': [u''], u'fq': [u''], u'limit': [u'100']}>,
POST:<QueryDict: {}>,
COOKIES:{'_ga': 'GA1.2.1639004996.1517425836',
'_gat_gtag_UA_68942400_4': '1',
'_gid': 'GA1.2.196350427.1519054822',
'csrftoken': <redacted>,
'sessionid': <redacted>},
META:{'CONTEXT_DOCUMENT_ROOT': '/var/www',
'CONTEXT_PREFIX': '',
u'CSRF_COOKIE': <redacted>,
'DOCUMENT_ROOT': '/var/www',
'GATEWAY_INTERFACE': 'CGI/1.1',
'HTTP_ACCEPT': 'application/json, text/plain, */*',
'HTTP_ACCEPT_ENCODING': 'gzip, deflate, br',
'HTTP_ACCEPT_LANGUAGE': 'en-US,en;q=0.9',
'HTTP_CONNECTION': 'keep-alive',
'HTTP_COOKIE': '_ga=GA1.2.1639004996.1517425836; _gid=GA1.2.196350427.1519054822; csrftoken=<redacted>; sessionid=<redacted>; _gat_gtag_UA_68942400_4=1',
'HTTP_HOST': 'beta.stemcellcommons.org',
'HTTP_REFERER': 'https://beta.stemcellcommons.org/files/',
'HTTP_USER_AGENT': 'Mozilla/5.0 (Macintosh; Intel Mac OS X 10_13_3) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/64.0.3282.167 Safari/537.36',
'HTTP_X_CSRFTOKEN': 'E4tuLKTD4Qb20dczzQEvtoj6QVmFcAV5',
'HTTP_X_FORWARDED_FOR': '134.174.140.132',
'HTTP_X_FORWARDED_PORT': '443',
'HTTP_X_FORWARDED_PROTO': 'https',
'PATH_INFO': u'/api/v2/files/',
'PATH_TRANSLATED': '/srv/refinery-platform/refinery/config/wsgi_aws.py/api/v2/files/',
'QUERY_STRING': 'fq=&limit=100&sort=',
'REMOTE_ADDR': '172.31.10.35',
'REMOTE_PORT': '58836',
'REQUEST_METHOD': 'GET',
'REQUEST_SCHEME': 'http',
'REQUEST_URI': '/api/v2/files/?fq=&limit=100&sort=',
'SCRIPT_FILENAME': '/srv/refinery-platform/refinery/config/wsgi_aws.py',
'SCRIPT_NAME': u'',
'SCRIPT_URI': 'http://beta.stemcellcommons.org/api/v2/files/',
'SCRIPT_URL': '/api/v2/files/',
'SERVER_ADDR': '172.31.10.66',
'SERVER_ADMIN': '[no address given]',
'SERVER_NAME': 'beta.stemcellcommons.org',
'SERVER_PORT': '80',
'SERVER_PROTOCOL': 'HTTP/1.1',
'SERVER_SIGNATURE': '',
'SERVER_SOFTWARE': 'Apache/2.4.7 (Ubuntu)',
'mod_wsgi.application_group': 'beta.stemcellcommons.org|',
'mod_wsgi.callable_object': 'application',
'mod_wsgi.enable_sendfile': '0',
'mod_wsgi.handler_script': '',
'mod_wsgi.input_chunked': '0',
'mod_wsgi.listener_host': '',
'mod_wsgi.listener_port': '80',
'mod_wsgi.process_group': 'refinery',
'mod_wsgi.queue_start': '1519414196923058',
'mod_wsgi.request_handler': 'wsgi-script',
'mod_wsgi.script_reloading': '1',
'mod_wsgi.version': (3, 4),
'wsgi.errors': <mod_wsgi.Log object at 0x7f0109a86730>,
'wsgi.file_wrapper': <built-in method file_wrapper of mod_wsgi.Adapter object at 0x7f010a154e40>,
'wsgi.input': <mod_wsgi.Input object at 0x7f0109a86b70>,
'wsgi.multiprocess': False,
'wsgi.multithread': True,
'wsgi.run_once': False,
'wsgi.url_scheme': 'http',
'wsgi.version': (1, 0)}>
```

### Expected behavior
No error

### LINKED ISSUE #2729: Handle ValueError in insert_facet_field_filter
* Specific code commit: v1.6.3
* Version of the web browser and OS: N/A
* Environment where the error occurred: AWS

### Steps to reproduce
Unknown, error report received by email

### Observed behavior
Error log:
```
Internal Server Error: /api/v2/assays/9cce34a3-fc84-4b6e-8a2c-a0448f267126/files/
Traceback (most recent call last):
  File "/home/ubuntu/.virtualenvs/refinery-platform/lib/python2.7/site-packages/django/core/handlers/base.py", line 132, in get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "/home/ubuntu/.virtualenvs/refinery-platform/lib/python2.7/site-packages/django/views/decorators/csrf.py", line 58, in wrapped_view
    return view_func(*args, **kwargs)
  File "/home/ubuntu/.virtualenvs/refinery-platform/lib/python2.7/site-packages/django/views/generic/base.py", line 71, in view
    return self.dispatch(request, *args, **kwargs)
  File "/home/ubuntu/.virtualenvs/refinery-platform/lib/python2.7/site-packages/rest_framework/views.py", line 466, in dispatch
    response = self.handle_exception(exc)
  File "/home/ubuntu/.virtualenvs/refinery-platform/lib/python2.7/site-packages/rest_framework/views.py", line 463, in dispatch
    response = handler(request, *args, **kwargs)
  File "/srv/refinery-platform/refinery/data_set_manager/views.py", line 812, in get
    solr_params = generate_solr_params_for_assay(params, uuid)
  File "/srv/refinery-platform/refinery/data_set_manager/utils.py", line 627, in generate_solr_params_for_assay
    return generate_solr_params(params, [assay_uuid], False, exclude_facets)
  File "/srv/refinery-platform/refinery/data_set_manager/utils.py", line 695, in generate_solr_params
    facet_field = insert_facet_field_filter(facet_filter, facet_field)
  File "/srv/refinery-platform/refinery/data_set_manager/utils.py", line 744, in insert_facet_field_filter
    facet_filter = json.loads(facet_filter)
  File "/usr/lib/python2.7/json/__init__.py", line 338, in loads
    return _default_decoder.decode(s)
  File "/usr/lib/python2.7/json/decoder.py", line 366, in decode
    obj, end = self.raw_decode(s, idx=_w(s, 0).end())
  File "/usr/lib/python2.7/json/decoder.py", line 384, in raw_decode
    raise ValueError("No JSON object could be decoded")
ValueError: No JSON object could be decoded

Request repr():
<WSGIRequest
path:/api/v2/assays/9cce34a3-fc84-4b6e-8a2c-a0448f267126/files/,
GET:<QueryDict: {u'data_set_uuid': [u'8566395e-a41e-4f49-89c2-bd52444a4707'], u'offset': [u'0'], u'limit': [u'100'], u'filter_attribute': [u'{"REFINERY_ANALYSIS_UUID_160_80_s":TF ChIP-Seq analysis using MACS2: hg19 2016-7-22@2:28:54']}>,
POST:<QueryDict: {}>,
COOKIES:{'_ga': 'GA1.2.1354209348.1507481536',
'_gid': 'GA1.2.1215373137.1524486035',
'csrftoken': <redacted>,
'sessionid': <redacted>},
META:{'CONTEXT_DOCUMENT_ROOT': '/var/www',
'CONTEXT_PREFIX': '',
u'CSRF_COOKIE': <redacted>,
'DOCUMENT_ROOT': '/var/www',
'GATEWAY_INTERFACE': 'CGI/1.1',
'HTTP_ACCEPT': 'text/html,application/xhtml+xml,application/xml;q=0.9,image/webp,image/apng,*/*;q=0.8',
'HTTP_ACCEPT_ENCODING': 'gzip, deflate, br',
'HTTP_ACCEPT_LANGUAGE': 'en-US,en;q=0.9',
'HTTP_CONNECTION': 'keep-alive',
'HTTP_COOKIE': '_ga=GA1.2.1354209348.1507481536; csrftoken=<redacted>; sessionid=<redacted>; _gid=GA1.2.1215373137.1524486035',
'HTTP_HOST': 'beta.stemcellcommons.org',
'HTTP_UPGRADE_INSECURE_REQUESTS': '1',
'HTTP_USER_AGENT': 'Mozilla/5.0 (Macintosh; Intel Mac OS X 10_11_6) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/65.0.3325.181 Safari/537.36',
'HTTP_X_FORWARDED_FOR': '209.6.122.26',
'HTTP_X_FORWARDED_PORT': '443',
'HTTP_X_FORWARDED_PROTO': 'https',
'PATH_INFO': u'/api/v2/assays/9cce34a3-fc84-4b6e-8a2c-a0448f267126/files/',
'PATH_TRANSLATED': '/srv/refinery-platform/refinery/config/wsgi_aws.py/api/v2/assays/9cce34a3-fc84-4b6e-8a2c-a0448f267126/files/',
'QUERY_STRING': 'data_set_uuid=8566395e-a41e-4f49-89c2-bd52444a4707&filter_attribute=%7B%22REFINERY_ANALYSIS_UUID_160_80_s%22:TF%20ChIP-Seq%20analysis%20using%20MACS2:%20hg19%202016-7-22@2:28:54&limit=100&offset=0',
'REMOTE_ADDR': '172.30.0.35',
'REMOTE_PORT': '42972',
'REQUEST_METHOD': 'GET',
'REQUEST_SCHEME': 'http',
'REQUEST_URI': '/api/v2/assays/9cce34a3-fc84-4b6e-8a2c-a0448f267126/files/?data_set_uuid=8566395e-a41e-4f49-89c2-bd52444a4707&filter_attribute=%7B%22REFINERY_ANALYSIS_UUID_160_80_s%22:TF%20ChIP-Seq%20analysis%20using%20MACS2:%20hg19%202016-7-22@2:28:54&limit=100&offset=0',
'SCRIPT_FILENAME': '/srv/refinery-platform/refinery/config/wsgi_aws.py',
'SCRIPT_NAME': u'',
'SCRIPT_URI': 'http://beta.stemcellcommons.org/api/v2/assays/9cce34a3-fc84-4b6e-8a2c-a0448f267126/files/',
'SCRIPT_URL': '/api/v2/assays/9cce34a3-fc84-4b6e-8a2c-a0448f267126/files/',
'SERVER_ADDR': '172.30.0.50',
'SERVER_ADMIN': '[no address given]',
'SERVER_NAME': 'beta.stemcellcommons.org',
'SERVER_PORT': '80',
'SERVER_PROTOCOL': 'HTTP/1.1',
'SERVER_SIGNATURE': '',
'SERVER_SOFTWARE': 'Apache/2.4.7 (Ubuntu)',
'mod_wsgi.application_group': 'beta.stemcellcommons.org|',
'mod_wsgi.callable_object': 'application',
'mod_wsgi.enable_sendfile': '0',
'mod_wsgi.handler_script': '',
'mod_wsgi.input_chunked': '0',
'mod_wsgi.listener_host': '',
'mod_wsgi.listener_port': '80',
'mod_wsgi.process_group': 'refinery',
'mod_wsgi.queue_start': '1524611205549539',
'mod_wsgi.request_handler': 'wsgi-script',
'mod_wsgi.script_reloading': '1',
'mod_wsgi.version': (3, 4),
'wsgi.errors': <mod_wsgi.Log object at 0x7f0456493470>,
'wsgi.file_wrapper': <built-in method file_wrapper of mod_wsgi.Adapter object at 0x7f04565c2120>,
'wsgi.input': <mod_wsgi.Input object at 0x7f0456493230>,
'wsgi.multiprocess': False,
'wsgi.multithread': True,
'wsgi.run_once': False,
'wsgi.url_scheme': 'http',
'wsgi.version': (1, 0)}>
```

### Expected behavior
No error

### Notes
See also #2710 and #2931

### LINKED ISSUE #2931: Handle ValueError in generate_solr_params
* Specific code commit: v1.6.5
* Version of the web browser and OS: unknown
* Environment where the error occurred: AWS

### Steps to reproduce
Unknown, error report received by email

### Observed behavior
```
Internal Server Error: /api/v2/assays/bf4a3bbc-6bdd-402a-a849-bc42b42dda30/files/
Traceback (most recent call last):
  File "/home/ubuntu/.virtualenvs/refinery-platform/lib/python2.7/site-packages/django/core/handlers/base.py", line 132, in get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "/home/ubuntu/.virtualenvs/refinery-platform/lib/python2.7/site-packages/django/views/decorators/csrf.py", line 58, in wrapped_view
    return view_func(*args, **kwargs)
  File "/home/ubuntu/.virtualenvs/refinery-platform/lib/python2.7/site-packages/django/views/generic/base.py", line 71, in view
    return self.dispatch(request, *args, **kwargs)
  File "/home/ubuntu/.virtualenvs/refinery-platform/lib/python2.7/site-packages/rest_framework/views.py", line 466, in dispatch
    response = self.handle_exception(exc)
  File "/home/ubuntu/.virtualenvs/refinery-platform/lib/python2.7/site-packages/rest_framework/views.py", line 463, in dispatch
    response = handler(request, *args, **kwargs)
  File "/srv/refinery-platform/refinery/data_set_manager/views.py", line 862, in get
    solr_params = generate_solr_params_for_assay(params, uuid)
  File "/srv/refinery-platform/refinery/data_set_manager/utils.py", line 629, in generate_solr_params_for_assay
    return generate_solr_params(params, [assay_uuid], False, exclude_facets)
  File "/srv/refinery-platform/refinery/data_set_manager/utils.py", line 715, in generate_solr_params
    facet_filter = json.loads(facet_filter)
  File "/usr/lib/python2.7/json/__init__.py", line 338, in loads
    return _default_decoder.decode(s)
  File "/usr/lib/python2.7/json/decoder.py", line 366, in decode
    obj, end = self.raw_decode(s, idx=_w(s, 0).end())
  File "/usr/lib/python2.7/json/decoder.py", line 382, in raw_decode
    obj, end = self.scan_once(s, idx)
ValueError: Invalid \escape: line 1 column 87 (char 86)

Request repr():
<WSGIRequest
path:/api/v2/assays/bf4a3bbc-6bdd-402a-a849-bc42b42dda30/files/,
GET:<QueryDict: {u'data_set_uuid': [u'ce201ea2-2705-4dc1-9f0e-4a315bacb9ca'], u'sort': [u'REFINERY_ANALYSIS_UUID_940_474_s desc'], u'offset': [u'100'], u'limit': [u'100'], u'filter_attribute': [u'{"REFINERY_NAME_940_474_s":["int_mes_CD31%2BLYVE-_con1%2C%5C%5Crc-stor4.dfci.harvard.edu%5Crslab%5C1%5Cdata_store%5CNeil%5Cneil_rnaseq_6-13-17%5Cfastq%5CCD31_CON_1_26_17.fastq.gz%2CMouse%2CN%2FA%2CRNA-Seq%2Cmm10%2C6%2F13%2F2017%2CNeil%2CWT%2CCD31%2FLYVE1%2Cendothelial%2C"]}']}>,
POST:<QueryDict: {}>,
COOKIES:{'_ga': 'GA1.2.1782439667.1533048077',
'_gid': 'GA1.2.998494014.1533311206',
'csrftoken': <redacted>,
'sessionid': <redacted>},
META:{'CONTEXT_DOCUMENT_ROOT': '/var/www',
'CONTEXT_PREFIX': '',
u'CSRF_COOKIE': <redacted>,
'DOCUMENT_ROOT': '/var/www',
'GATEWAY_INTERFACE': 'CGI/1.1',
'HTTP_ACCEPT': 'application/json, text/plain, */*',
'HTTP_ACCEPT_ENCODING': 'gzip, deflate, br',
'HTTP_ACCEPT_LANGUAGE': 'en-US,en;q=0.9',
'HTTP_CONNECTION': 'keep-alive',
'HTTP_COOKIE': '_ga=GA1.2.1782439667.1533048077; csrftoken=<redacted>; sessionid=<redacted>; _gid=GA1.2.998494014.1533311206',
'HTTP_HOST': 'beta.stemcellcommons.org',
'HTTP_REFERER': 'https://beta.stemcellcommons.org/data_sets/ce201ea2-2705-4dc1-9f0e-4a315bacb9ca/',
'HTTP_USER_AGENT': 'Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/67.0.3396.99 Safari/537.36',
'HTTP_X_CSRFTOKEN': <redacted>,
'HTTP_X_FORWARDED_FOR': '170.223.207.85',
'HTTP_X_FORWARDED_PORT': '443',
'HTTP_X_FORWARDED_PROTO': 'https',
'PATH_INFO': u'/api/v2/assays/bf4a3bbc-6bdd-402a-a849-bc42b42dda30/files/',
'PATH_TRANSLATED': '/srv/refinery-platform/refinery/config/wsgi_aws.py/api/v2/assays/bf4a3bbc-6bdd-402a-a849-bc42b42dda30/files/',
'QUERY_STRING': 'data_set_uuid=ce201ea2-2705-4dc1-9f0e-4a315bacb9ca&filter_attribute=%7B%22REFINERY_NAME_940_474_s%22:%5B%22int_mes_CD31%252BLYVE-_con1%252C%255C%255Crc-stor4.dfci.harvard.edu%255Crslab%255C1%255Cdata_store%255CNeil%255Cneil_rnaseq_6-13-17%255Cfastq%255CCD31_CON_1_26_17.fastq.gz%252CMouse%252CN%252FA%252CRNA-Seq%252Cmm10%252C6%252F13%252F2017%252CNeil%252CWT%252CCD31%252FLYVE1%252Cendothelial%252C%22%5D%7D&limit=100&offset=100&sort=REFINERY_ANALYSIS_UUID_940_474_s+desc',
'REMOTE_ADDR': '172.30.0.109',
'REMOTE_PORT': '57290',
'REQUEST_METHOD': 'GET',
'REQUEST_SCHEME': 'http',
'REQUEST_URI': '/api/v2/assays/bf4a3bbc-6bdd-402a-a849-bc42b42dda30/files/?data_set_uuid=ce201ea2-2705-4dc1-9f0e-4a315bacb9ca&filter_attribute=%7B%22REFINERY_NAME_940_474_s%22:%5B%22int_mes_CD31%252BLYVE-_con1%252C%255C%255Crc-stor4.dfci.harvard.edu%255Crslab%255C1%255Cdata_store%255CNeil%255Cneil_rnaseq_6-13-17%255Cfastq%255CCD31_CON_1_26_17.fastq.gz%252CMouse%252CN%252FA%252CRNA-Seq%252Cmm10%252C6%252F13%252F2017%252CNeil%252CWT%252CCD31%252FLYVE1%252Cendothelial%252C%22%5D%7D&limit=100&offset=100&sort=REFINERY_ANALYSIS_UUID_940_474_s+desc',
'SCRIPT_FILENAME': '/srv/refinery-platform/refinery/config/wsgi_aws.py',
'SCRIPT_NAME': u'',
'SCRIPT_URI': 'http://beta.stemcellcommons.org/api/v2/assays/bf4a3bbc-6bdd-402a-a849-bc42b42dda30/files/',
'SCRIPT_URL': '/api/v2/assays/bf4a3bbc-6bdd-402a-a849-bc42b42dda30/files/',
'SERVER_ADDR': '172.30.0.216',
'SERVER_ADMIN': '[no address given]',
'SERVER_NAME': 'beta.stemcellcommons.org',
'SERVER_PORT': '80',
'SERVER_PROTOCOL': 'HTTP/1.1',
'SERVER_SIGNATURE': '',
'SERVER_SOFTWARE': 'Apache/2.4.7 (Ubuntu)',
'mod_wsgi.application_group': 'beta.stemcellcommons.org|',
'mod_wsgi.callable_object': 'application',
'mod_wsgi.enable_sendfile': '0',
'mod_wsgi.handler_script': '',
'mod_wsgi.input_chunked': '0',
'mod_wsgi.listener_host': '',
'mod_wsgi.listener_port': '80',
'mod_wsgi.process_group': 'refinery',
'mod_wsgi.queue_start': '1533311530997217',
'mod_wsgi.request_handler': 'wsgi-script',
'mod_wsgi.script_reloading': '1',
'mod_wsgi.version': (3, 4),
'wsgi.errors': <mod_wsgi.Log object at 0x7f2afe87fcf0>,
'wsgi.file_wrapper': <built-in method file_wrapper of mod_wsgi.Adapter object at 0x7f2afebd45d0>,
'wsgi.input': <mod_wsgi.Input object at 0x7f2afe87ff70>,
'wsgi.multiprocess': False,
'wsgi.multithread': True,
'wsgi.run_once': False,
'wsgi.url_scheme': 'http',
'wsgi.version': (1, 0)}>
```

### Expected behavior
Error is handled

### Notes
See also #2729 and #2710

### PR DESCRIPTION
**Pull request by jkmarx**
Resolves #2623 
Resolves #2931 
Resolves #2729 (Remove insert_facet_field method. No longer needed.)

### CODE REVIEW

File: refinery/data_set_manager/utils.py [hunk 1]
```diff
@@ -635,91 +631,108 @@ def generate_solr_params(
...
+                )
+            factor_str = '{}_Factor_Value{}'.format(
+                    facet,
+                    NodeIndex.GENERIC_SUFFIX
+                )
+
+            if not facet_field:
+                facet_field = ','.join([char_str, factor_str])
+            else:
+                facet_field = ','.join([facet_field, char_str, factor_str])
```
Comment (mccalluc) [inline refinery/data_set_manager/utils.py]: Maybe build up a list and join it once at the end instead?

File: refinery/data_set_manager/utils.py [hunk 2]
```diff
@@ -926,24 +913,28 @@ def format_solr_response(solr_response):
...
+        count_array = count_obj.get('buckets')
+        count_obj_array = []
+        for field_obj in count_array:
             count_obj_array.append(
-                    {'name': count_array[index],
-                     'count': count_array[index + 1]
-                     })
+                    {'name': field_obj.get('val'),
+                     'count': field_obj.get('count')}
+                    )
```
Comment (mccalluc) [inline refinery/data_set_manager/utils.py]: List comprehension clearer?

File: refinery/data_set_manager/tests.py [hunk 3]
```diff
@@ -41,13 +42,12 @@
...
-                    format_solr_response, generate_facet_fields_query,
-                    generate_filtered_facet_fields,
+                    format_solr_response, generate_filtered_facet_fields,
                     generate_solr_params_for_assay,
                     get_file_url_from_node_uuid, get_owner_from_assay,
                     hide_fields_from_list, initialize_attribute_order_ranks,
-                    insert_facet_field_filter, is_field_in_hidden_list,
-                    objectify_facet_field_counts, update_annotated_nodes,
+                    is_field_in_hidden_list, update_annotated_nodes,
+                    create_facet_field_counts,
```
[the remark this answers is not in the document]
Comment (jkmarx) [reply refinery/data_set_manager/tests.py]: Yes to worry. Thanks

File: refinery/data_set_manager/utils.py (hunk 1 above)
Comment (jkmarx) [reply refinery/data_set_manager/utils.py]: I was doing other things here before, I'll just revert to the previous code now.

**Pull request merged**